client/viame-web-common/components/Viewer.vue

```diff
@@ -118,13 +118,15 @@ export default defineComponent({
       new HeadTail(),
     ];
 
+    const markMetaChangesPending = () => markChangesPending('meta');
```

Comment:
This is just a personal opinion related to organization so feel free to push back.  This might not be necessary because `markChangesPending` already defaults to 'meta' and anytime it is used in `useStyling` or `useTrackFilters` it is called without an argument anyways.  There may be uses cases I'm not thinking of with users of `vue-media-annotator` though.

I'm just thinking in the perspective of someone who isn't familiar with the system and would have to examine the flow of data to see what is happening.  If they were to add another hypothetical category besides `upsert | delete | meta` that is changed in  `useStyling` or `useTrackFilters` like lets stay `typeConfidence`.  So you add some calls to `markChangesPending('typeConfidence')` in `useTrackFilters` but it doesn't work because the outside function is overriding it to 'meta'.  You need to look up and understand that all calls within the system to `markChangesPending` are not the same.

Reply:
You're absolutely right.  I'll change it.